.cloudbees/workflows/workflow.yaml

```diff
@@ -16,13 +20,15 @@ jobs:
       with:
         entrypoint: /opt/bats/bin/bats
         args: .
-    - id: dockerconfig
```

Comment:
I strongly advise including a step to verify the testing action is the same as the release one, e.g. https://github.com/cloudbees-io/configure-ecr-credentials/pull/6/files#diff-d2f329414230659bb47c3d518cf6a3124abbb8e5846408af23d3dff3af195c51R21-R29 is one example of such a step